src/index.html

```diff
@@ -11,6 +11,35 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      data-qa="big"
+      class="search-form search-form--big"
+      action="/"
+      method="get">
```

Comment:
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/84296041/f400e3ab-18a0-4de9-82a2-07c538f47f1a)

```suggestion
    <form
      data-qa="big"
      class="search-form search-form--big"
      action="/"
      method="get"
    >
```